runtime/criusupport/criusupport.cpp

```diff
@@ -245,11 +247,26 @@ getNativeString(J9VMThread *currentThread, j9object_t javaString, char **nativeS
 	return res;
 }
 
+/**
+ * Checks if a thread is used by JDWP for debugging. Use string compare to check that the thread
+ * name starts with "JDWP" since all JDWP threads have that as a prefix.
+ *
+ * @param[in] currentThread the thread being checked
+ *
+ * @return true if the thread is used by JDWP, false if not
+ */
+static bool
+isJdwpDebugThread(J9VMThread *currentThread)
+{
+	char *threadName = getOMRVMThreadName(currentThread->omrVMThread);
+	return 0 == strncmp("JDWP", threadName, 4);
+}
+
 /**
  * Caller must first acquire exclusive safepoint or exclusive VMAccess
  */
 static void
-toggleSuspendOnJavaThreads(J9VMThread *currentThread, BOOLEAN suspend)
+toggleSuspendOnJavaThreads(J9VMThread *currentThread, BOOLEAN suspend, BOOLEAN jdwpSpecialAction)
```

Comment:
perhaps  `toggleDebuggingThreads` is more descriptive than `jdwpSpecialAction`

Reply:
Changed in https://github.com/eclipse-openj9/openj9/pull/16653/commits/15a2fc49467422d926e8a58659658e35876c4e14